client/packages/host/src/CommandK.tsx

```diff
@@ -258,50 +245,47 @@ const Actions = () => {
     },
     {
       id: 'action:logout',
-      name: `${t('logout')} (Ctrl+L)`,
-      shortcut: ['$mod+KeyL'],
+      name: `${t('logout')} (Alt/Option+Shift+L)`,
+      shortcut: ['Alt+Shift+KeyL'],
       keywords: 'logout',
       perform: () => confirmLogout({}),
     },
     {
       id: 'action:easter-egg',
-      name: `${t('easter-egg')}`,
-      shortcut: ['$mod+Shift+KeyE'],
+      name: `${t('easter-egg')} (Alt/Option+Shift+E)`,
+      shortcut: ['Alt+Shift+KeyE'],
       keywords: 'easter egg game',
       perform: showEasterEgg,
     },
     {
       id: 'navigation:help',
-      name: `${t('help')} (Alt+H)`,
+      name: `${t('help')} (Alt/Option+H)`,
       keywords: 'help, docs, guide',
       shortcut: ['Alt+KeyH'],
       perform: () => navigate(RouteBuilder.create(AppRoute.Help).build()),
     },
     {
       id: 'action:sync',
-      name: `${t('sync')} (Alt+Control+S)`,
+      name: `${t('sync')} (Alt/Option+Shift+S)`,
       keywords: 'sync',
-      shortcut: ['Alt+$mod+KeyS'],
+      shortcut: ['Alt+Shift+KeyS'],
       perform: showSync,
     },
   ];
 
-  if (userHasPermission(UserPermission.ServerAdmin)) {
```

Comment:
This should have been changed when the settings section was opened up to all users